components/azure/azureml/aml-deploy-model/DockerFile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/azure-cli
+RUN az extension add -n azure-cli-ml

```

Comment:
Combine multiple runs into one and perform pip upgrade before installation of az-cli-ml

Reply:
Changed